packages/public/server/__tests-e2e__/login.ts

```diff
@@ -19,49 +19,122 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
  */
-import { pages, viewports, users, elements } from './_config'
+import { pages, viewports, elements, testingServerUrl } from './_config'
 import {
   clickForNewPage,
   getByPlaceholderText,
   getBySelector,
   getByText,
+  getByRole,
   goto,
 } from './_utils'
+import { ElementHandle } from 'puppeteer'
 
-const examplePages = ['/', '/math', '/math/geometry/triangles']
+test('correct password and correct user', async () => {
+  await page.setViewport(viewports.desktop)
+  const firstPage = await goto('/math')
```

Comment:
```suggestion
beforeAll(async () => {
  await page.setViewport(viewports.desktop)
})

test('correct password and correct user', async () => {
  const firstPage = await goto('/math')
```

Reply:
So you can delete the `page.setViewport(viewports.desktop)` in the other test cases...